reference/OpenAPI.json

```diff
@@ -66936,7 +67041,7 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/8_res_create_reward_assignment"
+                  "$ref": "#/components/schemas/LoyaltiesRewardsCreateAssignmentRequestBody"
```

Comment:
Schemat `RequestBody`, ale tu jest `responses` i to jest jedyne użycie tego schematu. Chyba tutaj miał wylądować `LoyaltiesRewardsCreateAssignmentResponseBody`.

Ta ścieżka `"/v1/loyalties/{campaignId}/rewards"` jest zrobiona inaczej, niż większość endpointów, czyli najpierw idzie response, a potem request. Stąd pewnie ta pomyłka.